lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -105,6 +105,12 @@ module Resources
             'status': 'accepted'
           }
         ],
+        'patientInstruction': "Preparations:\n- boop\n- Arrive 15 minutes early\n- Can't be empty\n- comment" \
+                              "\nDirections:\n- comment\n- Do not drive after appointment. \r\n\r\nConsider " \
+                              "Bringing a Family Member or Friend to the Doctor's Office. If that is not possible, " \
+                              "the Reception desk can call a cab for you.\nPost Appointment Instructions:\n" \
+                              "- Post Appointment\n- Post Order\n- Be Safe while driving back after visit. Be aware " \
+                              "of white walkers.\n- Post Order\n- comment\nPost Appointment Instructions.:\n- Comment",
```

Comment:
Are these accurate instructions to what someone would see in sandbox/prod? We should keep these realistic since it's public facing.

Reply:
Picked the new value from prod:
https://github.com/cerner/fhir.cerner.com/commit/36a80da88afe0e26ade159eb7c127d6663da7ff3#diff-e2510815c297dacadd55ed896d98b2008eac6a19cd89b6a31bfc34ae3074165dR108